python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_checks.py

```diff
@@ -575,6 +579,123 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
     assert result.success
 
 
+def test_asset_checks_evaluations(
+    test_asset_checks_manifest: dict[str, Any], dbt_commands: list[list[str]]
+):
+    @op
+    def my_dbt_op(context: OpExecutionContext, dbt: DbtCliResource):
+        events = []
+        invocation_id = ""
+        for dbt_command in dbt_commands:
+            dbt_invocation = dbt.cli(
+                dbt_command,
+                context=context,
+                raise_on_error=False,
+                manifest=test_asset_checks_manifest,
+                dagster_dbt_translator=dagster_dbt_translator_with_checks,
+            )
+            events += list(dbt_invocation.stream())
+            invocation_id = dbt_invocation.get_artifact("run_results.json")["metadata"][
+                "invocation_id"
+            ]
+
+        for event in events:
+            if isinstance(event, AssetCheckEvaluation):
+                assert cast(int, event.metadata["Execution Duration"].value) > 0
+
+        expected_results = [
+            AssetCheckResult(
```

Comment:
Why `AssetCheckResult` here instead of `AssetCheckEvaluation`? Actually a bit confused as to why the test would pass, maybe because they're both namedtuples with the same set of fields?

Reply:
Good catch - it should have been `AssetCheckEvaluation`. Like you said, tests were passing because they're both namedtuples with the same set of fields. I added another assertion to make sure there is no `AssetCheckResult` in the events.